basis_set_exchange/data/METADATA.json

```diff
@@ -34824,6 +34924,52 @@
       }
     }
   },
+  "jorge-tzp-zora": {
+    "display_name": "jorge-TZP-ZORA",
+    "other_names": [],
+    "description": "TZP-ZORA: All-electron triple zeta valence quality plus polarization functions (ZORA)",
+    "latest_version": "1",
+    "tags": [],
+    "basename": "jorge-TZP-ZORA",
+    "relpath": "",
+    "family": "jorge",
+    "role": "orbital",
+    "function_types": [
+      "gto",
+      "gto_cartesian"
```

Comment:
this looks odd, I think these should be spherical. I don't see the sources in the basis_set_exchange-historical repo

Reply:
I can double check the paper. The basis sets emailed to me were in nwchem format and missing SPHERICAL in the top line. I would assume they are spherical too

Reply:
If they used NWChem, then the sets are cartesian...

Reply:
Skimming the papers, looks like they used ORCA for the calculations, but they sent the basis sets in NWChem format. Some of their previous basis sets were also Gaussian format.